### PR TITLE
Feat responsive design

### PR DESCRIPTION
**Pull request by kkatusic**
- added responsive classes for homepage
- added mobile menu
- added menu switcher

### CODE REVIEW

File: apps/platform/src/app/components/LanguageSwitcher.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		screen === 'mobile'
+			? 'border-borderGreenDark w-full relative inline-block text-left'
+			: 'relative inline-block text-left';
+
+	const classNameToggler =
+		screen === 'mobile'
+			? 'flex text-white w-full bg-white text-xs text-[#999999] justify-between rounded-md px-3 py-2 items-center mb-6'
+			: 'border-green inline-flex items-center justify-center rounded-md border px-3 py-2 text-base font-normal text-green focus:outline-none';
+
+	const swithHolder =
```
Comment (b0rza) [inline apps/platform/src/app/components/LanguageSwitcher.tsx:41]: /nitpick
```suggestion
	const switchHolder =
```

File: apps/platform/src/app/layout.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import type { Metadata } from 'next';
+import type { Viewport } from 'next'
```
Comment (b0rza) [inline apps/platform/src/app/layout.tsx:2]: Unused?

**Review by b0rza — CHANGES_REQUESTED** [review]
gg, some nitpicks 👍

File: apps/platform/src/app/components/LanguageSwitcher.tsx (hunk 1 above)
Comment (kkatusic) [reply apps/platform/src/app/components/LanguageSwitcher.tsx:41]: resolved

File: apps/platform/src/app/layout.tsx (hunk 2 above)
Comment (kkatusic) [reply apps/platform/src/app/layout.tsx:2]: resolved

**Review by b0rza — APPROVED** [review]

**Pull request merged**